tests/e2e/etcd_config_test.go

```diff
@@ -21,7 +21,9 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
 	"go.etcd.io/etcd/pkg/v3/expect"
+	"go.etcd.io/etcd/tests/v3/framework/e2e"
```

Comment:
@ahrtr I've hit a bit of a branching point with this backport here.  The tests structure between 3.5 and 3.6 is a bit different so this import and one other in `v3_tls_test.go` don't work.

Can you please provide some guidance on how you would approach resolving the differences from here?

Reply:
There is no test framework in release-3.5 and 3.4. You can follow the existing test cases, please also see my comments below.

Reply:
Thanks @ahrtr, makes sense. I wasn't sure if there was going to be a requirement to backport any of the test framework and wanted to check 🙏🏻